src/main/java/org/apache/commons/lang3/time/DurationFormatUtils.java

```diff
@@ -171,56 +175,96 @@ public static String formatDuration(final long durationMillis, final String form
      * @throws java.lang.IllegalArgumentException if durationMillis is negative
      */
     public static String formatDurationWords(
-        final long durationMillis,
-        final boolean suppressLeadingZeroElements,
-        final boolean suppressTrailingZeroElements) {
-
-        // This method is generally replaceable by the format method, but
-        // there are a series of tweaks and special cases that require
-        // trickery to replicate.
-        String duration = formatDuration(durationMillis, "d' days 'H' hours 'm' minutes 's' seconds'");
+            final long durationMillis,
+            final boolean suppressLeadingZeroElements,
+            final boolean suppressTrailingZeroElements) {
+        return formatDurationWordsWithMs(durationMillis, false,
+                                         suppressLeadingZeroElements, suppressTrailingZeroElements);
+    }
+
+    /**
+     * <p>Formats an elapsed time into a pluralization correct string.</p>
+     * <p>Returns duration in format '0 days 0 hours 0 minutes 0 seconds 0 milliseconds'</p>
+     * It removes all leading time units with 0 value.
```

Comment:
Dangling text. Was it supposed to be within the previous `<p>`?

Reply:
aligned to previous <p>